i18n/.js/src/main/scala/io/udash/i18n/bindings/TranslationBinding.scala

```diff
@@ -2,32 +2,33 @@ package io.udash.i18n.bindings
 
 import com.avsystem.commons._
 import io.udash.bindings.Bindings
-import io.udash.i18n.Translated
+import io.udash.bindings.modifiers._
+import io.udash.i18n._
 import io.udash.logging.CrossLogging
 import org.scalajs.dom._
 import scalatags.JsDom.Modifier
 
 import scala.concurrent.Future
 import scala.util.{Failure, Success}
 
-private[i18n] class TranslationBinding(translation: Future[Translated], placeholder: Option[Element])
-  extends Modifier with Bindings with CrossLogging {
+private[i18n] final class TranslationBinding(
+  translation: Future[Translated],
+  placeholder: Option[Element],
+  rawHtml: Boolean
+) extends Modifier with Bindings with CrossLogging {
 
-  override def applyTo(t: Element): Unit =
-    if (translation.isCompleted && translation.value.get.isSuccess) {
-      t.appendChild(document.createTextNode(translation.value.get.get.string))
-    } else {
-      val holder: Node = placeholder.getOrElse(emptyStringNode())
-      t.appendChild(holder)
+  override def applyTo(t: Element): Unit = translation.value match {
+    case Some(Success(Translated(text))) =>
```

Comment:
I'd opt for removing this questionable optimization. With `onCompleteNow` it should be almost the same and the `holder` is just a reference here, it shouldn't prevent GC after the callback is executed.

Reply:
@ddworak I don't think that `onCompleteNow` here will be same when you have latency with backend.

Probably I do not understand how `onCompleteNow` works, but for me it executed as soon as future is complete (aka response from backend is here), and if we have latency, we will see placehoder instead nothing.

Have I missed something?

Reply:
Ok, I guess I've got your idea about `onCompleteNow`. Updated this.